docs/migration.md

```diff
@@ -0,0 +1,191 @@
+have been removed, so the following command line flag:
+
+```
+./prometheus -alertmanager.url=http://alertmanager:9093/
+```
+
+Would be replaced with something like the following in the `prometheus.yml`
+config file:
+
+```yml
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets:
+      - alertmanager:9093
+```
+
+You can also use all the usual Prothetheus service discovery integrations and
+relabeling in your Alertmanager configuration.  This snippet instructs
+Prometheus to search for Kubernetes pods, in the `default` namespace, with the
+label `name: alertmanager` and with a non-empty port.
+
+```yml
+alerting:
+  alertmanagers:
+  - kubernetes_sd_configs:
+      - role: pod
+    tls_config:
+      ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+    relabel_configs:
+    - source_labels: [__meta_kubernetes_pod_label_name]
+      regex: alertmanager
+      action: keep
+    - source_labels: [__meta_kubernetes_namespace]
+      regex: default
+      action: keep
+    - source_labels: [__meta_kubernetes_pod_container_port_number]
+      regex:
+      action: drop
+```
+
+## Recording rules and alerts
+
+The format for configuring alerting and recording rules has been changed to YAML.
+An example of a recording rule and alert in the old format:
+
+```
+job:request_duration_seconds:99percentile =
+  histogram_quantile(0.99, sum(rate(request_duration_seconds_bucket[1m])) by (le, job))
+
+ALERT FrontendRequestLatency
+  IF job:request_duration_seconds:99percentile{job="frontend"} > 0.1
+  FOR 5m
+  ANNOTATIONS {
+    summary = "High frontend request latency",
+  }
+```
+
+Would look like this:
+
+```yml
+groups:
+- name: example.rules
+  rules:
+  - record: job:request_duration_seconds:99percentile
+    expr: histogram_quantile(0.99, sum(rate(request_duration_seconds_bucket[1m]))
+      BY (le, job))
+  - alert: FrontendRequestLatency
+    expr: job:request_duration_seconds:99percentile{job="frontend"} > 0.1
+    for: 5m
+    annotations:
+      summary: High frontend request latency
+```
+
+To help with the change, the `promtool` tool has a mode to automate the rules conversion.  Given a `.rules` file, it will output a `.rules.yml` file in the
+new format. For example:
+
+```
+$ promtool update rules example.rules
+```
+
+## Storage
+
+The data format in Prometheus 2.0 has completely changed and is not backwards
+compatible with 1.8. To retain access to your historic monitoring data we recommend
+you run a non-scraping Prometheus 1.8.1 instance in parallel with your Prometheus 2.0.
+
+Your Prometheus 1.8 instance should be started with the following flags and an
+empty config file (`empty.yml`):
+
+```
+$ ./prometheus-1.8.1.linux-amd64/prometheus -web.listen-address ":9094" -config.file empty.yml
+```
+
+NB If you used [external labels]() in your Prometheus config, there need to be
```

Comment:
`NB` -> `NOTE: **NOTE**:` (the first `NOTE` will be picked up by the docs page parser and generate a little notice box like the `CAUTION` box in https://prometheus.io/docs/practices/naming/#labels